<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>commDialog</class>
 <widget class="QDialog" name="commDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="cbOpenStart">
     <property name="text">
      <string>Open on Startup</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
       <item>
        <widget class="QLabel" name="labelPorts">
         <property name="text">
          <string>Serial Port:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbListPorts"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
       <item>
        <widget class="QLabel" name="labelSpeed">
         <property name="text">
          <string>Speed:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbSpeed">
         <item>
          <property name="text">
           <string>4800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>38400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>57600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_8n1">
       <property name="text">
        <string>N-8-1 only</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbHwFlow">
       <property name="text">
        <string>Use hardware flow control</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gbLog">
     <property name="title">
      <string>LogFile</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,0">
        <item>
         <widget class="QLineEdit" name="editLogFile"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnSelectLogFile">
          <property name="text">
           <string>Select</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="cbAddTimeStemp">
        <property name="text">
         <string>Add Time Stemp</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,1,0">
        <item>
         <widget class="QCheckBox" name="cbSplitFile">
          <property name="text">
           <string>Split File when File size over</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sbFileSize"/>
        </item>
        <item>
         <widget class="QLabel" name="labelFileSize">
          <property name="text">
           <string>MB</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>commDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>commDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
